src/node_osrm.cpp

```diff
@@ -4,594 +4,507 @@
 #include <nan.h>
 
 // OSRM
+#include <osrm/route_parameters.hpp>
 #include <osrm/json_container.hpp>
 #include <osrm/libosrm_config.hpp>
 #include <osrm/osrm.hpp>
-#include <osrm/route_parameters.hpp>
+
+#include <boost/optional.hpp>
 
 // STL
 #include <iostream>
 #include <memory>
```

Comment:
Missing headers:

  - boost/assert.hpp
  - vector
  - utility (for move, forward)
  - algorithm (for copy)
  - iterator (for back_inserter)
  - exception
  - string